README

```diff
@@ -0,0 +1,44 @@
+beginnings of an api...
+
+a large part of the effort here has been around testing and coverage...
+
+the api endpoints:
+GET     /
+GET     /v1/users
+GET     /v1/users/:id
+POST    /v1/users
+PUT     /v1/users/:id
+DELETE  /v1/users/:id
+
+Options to start the app:
+1. run dynamo locally and just:
+    > npm i && node.app
+2. docker-compose up
+
+Options to test the app
+1. if running dynamo and the app locally:
+    > npm run test ./models/*spec* ./*spec*
+2. if looking to run them through docker:
+    > ./test.sh
+3. if you want to run some curl tests when the app is up, you can:
+    > ./test_with_curl
+
+Some highlights:
+    - the user model tests against a local dynamo instance
+    - each test starts with a clean db to help isolate each test case
+    - a coverage report is available using nyc
+    - some ideas for how metrics and logging might work are included in comments
+    - a basic circle ci config is present but not tested, if not spot on, think it's close.
+
+Code quirks?
+    - I know it's unpopular, but I like to use let instead of const..
+    - It's less characters and let's me keep things dynamic (like the language).
```

Comment:
it's mostly a combination of muscle memory, ocd and allowing for variable reuse to avoid having to come up with more variable names... it's just a quirk, I know it's sacrilege ;)

Reply:
PS: that is an awesome profile pic.. like for real.